swing/src/net/sf/openrocket/gui/print/FinMarkingGuide.java

```diff
@@ -23,17 +23,18 @@
 import net.sf.openrocket.rocketcomponent.ExternalComponent;
 import net.sf.openrocket.rocketcomponent.FinSet;
 import net.sf.openrocket.rocketcomponent.LaunchLug;
+import net.sf.openrocket.rocketcomponent.RailButton; // ADD IMPORT Rail Buttons
```

Comment:
I have additional fixes related to that issue ready, along with drawing the guide if there are tube fins, but, because the code is interrelated, I need this PR merged before I commit the nest PR.

Reply:
I have additional fixes related to that issue ready, along with drawing the guide if there are tube fins, but, because the code is interrelated, I need this PR merged before I commit the nest PR.

Reply:
In the next PR, which will include importing the tube fins, I was considering using "import net.sf.openrocket.rocketcomponent.*" instead of multiple import lines. Is that encouraged or discouraged in this context?

Reply:
My preference is always for individual imports of every needed class, since it's both more explicit and gives me a better idea of where to look for stuff in the codebase.

Reply:
👍🏻

Reply:
Indeed, generally bad practice to use wildcard imports